data/language/pl-PL.yml

```diff
@@ -1809,22 +1795,22 @@ strings:
   1765: '{COLOUR BLACK}Wartość Firmy'
   1766: '{COLOUR BLACK}Wartość Firmy ▼'
   1767: "{NEWLINE 1 2}{SPRITE}{NEWLINE 27 8}{STRINGID}"
-  1768: "{NEWLINE 0 3}{INT16_1DP}%        {NEWLINE 0 13}“{STRINGID}”"
-  1769: "{NEWLINE 0 3}{INT16_1DP}%{SPRITE 2325}{NEWLINE 0 13}“{STRINGID}”"
-  1770: "{NEWLINE 0 3}{INT16_1DP}%{SPRITE 2324}{NEWLINE 0 13}“{STRINGID}”"
+  1768: "{NEWLINE 0 3}{INT16_1DP}%{NEWLINE 0 13}{STRINGID}"
+  1769: "{NEWLINE 0 3}{INT16_1DP}%{SPRITE 2325}{NEWLINE 0 13}{STRINGID}"
+  1770: "{NEWLINE 0 3}{INT16_1DP}%{SPRITE 2324}{NEWLINE 0 13}{STRINGID}"
   1771: "{NEWLINE 0 8}{CURRENCY48}"
-  1772: Robotnik
-  1773: Inżynier
-  1774: Kierownik Ruchu
-  1775: Koordynator Przewozów
-  1776: Nadzorca Tras
-  1777: Dyrektor
-  1778: Naczelny Wykonawca
-  1779: Przewodniczący
-  1780: Prezes
-  1781: Potentat Transportowy
-  1782: '{INT16} firma'
-  1783: '{INT16} firmy'
+  1772: '"Robotnik"'
```

Comment:
Please put the quotation marks back in the original strings rather than hardcoding them here. Also, please take care to use `“` and `”` instead of `"`.